Libraries/Components/Navigation/NavigatorIOS.ios.js

```diff
@@ -50,6 +52,36 @@ class NavigatorTransitionerIOS extends React.Component {
   }
 }
 
+var SystemIconsLabels = {
+  done: true,
+  cancel: true,
+  edit: true,
+  save: true,
+  add: true,
+  'flexible-space': true,
+  'fixed-space': true,
+  compose: true,
+  reply: true,
+  action: true,
+  organize: true,
+  bookmarks: true,
+  search: true,
+  refresh: true,
+  stop: true,
+  camera: true,
+  trash: true,
+  play: true,
+  pause: true,
+  rewind: true,
+  'fast-forward': true,
+  undo: true,
+  redo: true,
+  'page-curl': true,
+};
+var SystemIcons = keyMirror(SystemIconsLabels);
+
+type BarButtonSystemIcon = $Enum<typeof SystemIconsLabels>;
```

Comment:
What is `BarButton`? Should this be something like `NavBarButtonSystemIcon`? or just `SystemIconButton` or similar?